### ISSUE TITLE
CLion and Rust debugging issue

### ISSUE THREAD

**Issue by snambi** [intellij-rust/intellij-rust#4494]
This is a CLion and Rust debugging issue.

When a debug point is set on a library (dependency crate), the debugger doesn't while running in debug mode.

## Environment

* **IntelliJ Rust plugin version:**
`0.2.106.2135-192`

* **Rust toolchain version:**
```
$rustc -V
rustc 1.38.0-nightly (6e310f2ab 2019-07-07)

$rustup -V
rustup 1.19.0 (2af131cf9 2019-09-08)
```

* **IDE name and version:** 
CLion 2019.2.3
```
Build #CL-192.6817.18, built on September 24, 2019
Runtime version: 11.0.4+10-b304.69 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.6
GC: ParNew, ConcurrentMarkSweep
Memory: 1987M
Cores: 12
Registry: run.processes.with.pty=TRUE
Non-Bundled Plugins: com.vladsch.idea.multimarkdown, org.toml.lang, org.rust.lang
```

* **Operating system:**
```
Mac OS Majave
Version 10.14.6 (18G103)
```

## Problem description
Debugger doesn't stop in break-points set in dependency crates defined in `Cargo.toml`

**Comment by mandrean** [comment]
Can confirm I have the same issue

*Intellij:* 2019.2.1
*Intellij-Rust:* 0.2.106.2135-192
*rustc:* 1.38.0

**Comment by foisonocean** [comment]
I got the same issue.
OS: macOS 10.11.6
rustc: 1.38.0 (625451e37 2019-09-23)
IDE: CLion 2019.2.5
intellij-rust: 0.2.108.2148-192

@snambi @mandrean 
Have you solved your problem?

**Comment by mandrean** [comment]
@ortem @foisonocean In my case, the issue was that the corporate “smart” AI-powered anti-virus system on my work laptop started blocking LLDB as a malignant process.

I contacted IT support and they put LLDB on a whitelist. 

Can confirm I don’t have the same issue on my private laptop!

**Comment by foisonocean** [comment]
I don't have any anti-virus softwares in my mac

**Comment by foisonocean** [comment]
Hi all, I finally fixed this by uninstall the latest CLion 2019.2.5 and installed CLion 2019.2.1, it seems this bug only happens with the latest CLion version.

PS: I also tried delete CLion config files in `~/Library/Preferences`, or reinstall CLion 2019.2.5, none of them works.
